### ISSUE TITLE
Labels

### ISSUE THREAD

**Issue by curquiza** [watzon/octokit.cr#4]
Hello!
I would like to do the labels methods.
Any tips or things to know before starting? 😄 I'm not sure to understand how the tests work in this package for example.

**Comment by watzon** [comment]
I don't believe I ever wrote tests, but you're free to add anything you wish. Just look at the way current methods are written and try to follow the same style. Thanks :)

**Comment by curquiza** [comment]
I'm really new in the crystal community, I'm not sure I'm able to add tests if there is no example.
Thanks for your reactivity, I'll do my best!
